index.bs

```diff
@@ -569,7 +579,8 @@ When this method is invoked, the user agent MUST execute the following algorithm
     : {{CollectedClientData/challenge}}
     :: The [=base64url encoding=] of |options|.{{MakeCredentialOptions/challenge}}
     : {{origin}}
-    :: The [=unicode serialization of an origin|unicode serialization=] of |rpId|
+    :: |effectiveDomain|
+    <!-- might need to have both {{origin}} and {{rpId}} in collectedClientData... ? -->
```

Comment:
hrrrrmmm. So if we only include `effectiveDomain` in the client data, then the RP doesn't get looped-back what the `rpId` calculated was. But it still can tell what the origin of the requesting page was, and of course the RP's software is _probably_ the one setting the `rpId` in the first place.

I believe this works from the RP's perspective, and clearly it's not necessary for the authenticator. The question is, I guess: is there any security impact to the `rpId` not being in the signed hash?

Not being signed means the RP can't rely on the `rpId` being accurate to its intentions. If it were forged (through a MITM), it would allow a forger to potentially cause a signature with a different private key, since the `rpId` is often mixed into the key wrapping functions. But the MITM could also replace the included/excluded keys, which accomplishes the same thing without regard to the `rpId`. (To what end, I don't know)

So I think then it's OK to not sign the `rpId`, but I suggest others think through this too.